src/terms/ewald.jl

```diff
@@ -1,6 +1,33 @@
+struct Ewald end
+(E::Ewald)(basis) = EwaldTerm(basis)
+
+struct EwaldTerm <: Term
+    basis::PlaneWaveBasis
+    E::Real # precomputed energy
+end
+function EwaldTerm(basis::PlaneWaveBasis)
+    # precompute Ewald energy
+    E = energy_ewald(basis.model)
+    EwaldTerm(basis, E)
+end
+
+term_name(T::EwaldTerm) = "Ewald"
+
+function ene_ham(T::EwaldTerm, Psi, occ; kwargs...)
+    hams = [NoopRFO(T.basis, kpoint) for kpoint in T.basis.kpoints]
+    (E=T.E, hams=hams)
+end
+
+
 using SpecialFunctions: erfc
 using ForwardDiff
 
```

Comment:
It's kind of redundant with the other one and the definition of `atoms`, no? I don't want to talk about pseudopotentials there because it's not the job of this one term. This discussion of how we split the terms should go in a broader description.

Reply:
Ok that is true. Just remove that sentence than.